doc/source/data/examples/pytorch_resnet_batch_prediction.ipynb

```diff
@@ -557,23 +585,37 @@
    "source": [
     "from PIL import Image\n",
     "\n",
-    "for image, prediction in zip(prediction_batch[\"original_image\"], prediction_batch[\"predicted_label\"]):\n",
+    "for image, prediction in zip(\n",
+    "    prediction_batch[\"original_image\"], prediction_batch[\"predicted_label\"]\n",
+    "):\n",
     "    img = Image.fromarray(image)\n",
     "    display(img)\n",
-    "    print(\"Label: \", prediction)"
+    "    print(\"Label: \", prediction)\n"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {
     "tags": []
    },
    "source": [
-    "If the samples look good, we can proceed with saving the results to an external storage, e.g., S3 or local disks. See [Ray Data Input/Output](https://docs.ray.io/en/latest/data/api/input_output.html) for all supported stoarges and file formats.\n",
+    "If the samples look good, we can proceed with saving the results to an external storage, e.g., S3 or local disks. See [Ray Data Input/Output](https://docs.ray.io/en/latest/data/api/input_output.html) for all supported storage and file formats.\n"
```

Comment:
Can we replace the link with https://docs.ray.io/en/latest/data/saving-data.html instead? Thanks